### PR TITLE
Add comments to test_token.py

### PR DESCRIPTION
**Pull request by pirapira**
This is a part of #357.

### CODE REVIEW

File: raiden_contracts/tests/test_token.py [hunk 1]
```diff
@@ -43,6 +47,8 @@ def test_approve_transfer(web3, custom_token, get_accounts):
 
 
 def test_token_transfer_funds(web3, custom_token, get_accounts, txn_gas):
+    """ transferFuhnds() should fail when the ETH balance of the contract is zero """
```
Comment (loredanacirstea) [inline raiden_contracts/tests/test_token.py]: typo `transferFunds`

Comment (LefterisJP) [inline raiden_contracts/tests/test_token.py]: /s/Fuhnds/Funds

File: raiden_contracts/tests/test_token.py [hunk 2]
```diff
@@ -60,8 +67,12 @@ def test_token_transfer_funds(web3, custom_token, get_accounts, txn_gas):
 
 
 def test_custom_token(custom_token, web3):
-    assert web3.eth.getCode(custom_token.address)
+    """ See custom_token.address contains some big code """
+
+    assert len(web3.eth.getCode(custom_token.address)) > 100
```
Comment (LefterisJP) [inline raiden_contracts/tests/test_token.py]: Why care about the size of the code?

Comment (pirapira) [reply raiden_contracts/tests/test_token.py]: In case of empty, '0x' is returned.  `assert '0x'` silently succeeds.

**Comment by pirapira** [comment]
Waiting for CI.

File: raiden_contracts/tests/test_token.py (hunk 2 above)
Comment (LefterisJP) [reply raiden_contracts/tests/test_token.py]: Ah good point @pirapira

Comment (LefterisJP) [reply raiden_contracts/tests/test_token.py]: In that case you should also change the docstring though -- as anyone who reads the code will have the same question I had.

Comment (hackaugusto) [reply raiden_contracts/tests/test_token.py]: shouldn't it decode the hex representation and check the length is larger then 0?

Comment (pirapira) [reply raiden_contracts/tests/test_token.py]: Maybe we also have the exact expected hex string.

**Pull request merged**